frontend/app/src/components/accounts/blockchain/XpubInput.vue

```diff
@@ -4,14 +4,7 @@ import { Blockchain } from '@rotki/common/lib/blockchain';
 import useVuelidate from '@vuelidate/core';
 import { helpers, required } from '@vuelidate/validators';
 import isEmpty from 'lodash/isEmpty';
-import { trimOnPaste } from '@/utils/event';
-import {
-  XpubPrefix,
-  type XpubType,
-  getKeyType,
-  getPrefix,
-  keyType
-} from '@/utils/xpub';
+import { XpubPrefix, type XpubType } from '@/utils/xpub';
```

Comment:
can't the `XpubPrefix` be removed too her?

Reply:
it is used as a type and enum in code... auto-import only exports it as an enum